packages/tokens/package.json

```diff
@@ -24,7 +24,7 @@
   "devDependencies": {
     "@types/fs-extra": "^11.0.3",
     "@types/lodash": "^4.14.200",
-    "@types/node": "^20.8.9",
+    "@types/node": "^22.8.7",
```

Comment:
@alex-ju if you prefer to have the changes to the token's `package.json` file in a separate PR let me know, I can easily do it. It was done mainly to keep the two pipelines in sync.

Reply:
no, I'm ok with them in the same PR – keeping dependencies in sync sounds good